app/src/main/res/layout/item_footer_timeline.xml

```diff
@@ -23,24 +23,32 @@
         android:paddingStart="@dimen/dp_8"
         android:paddingTop="@dimen/dp_4">
 
-        <TextView
-            android:id="@+id/text_comment"
+        <LinearLayout
```

Comment:
WARNING: [UseCompoundDrawables] This tag and its children can be replaced by one `<TextView/>` and a compound drawable
 A `LinearLayout` which contains an `ImageView` and a `TextView` can be more efficiently handled as a compound drawable (a single TextView, using the `drawableTop`, `drawableLeft`, `drawableRight` and/or `drawableBottom` attributes to draw one or more images adjacent to the text).

If the two widgets are offset from each other with margins, this can be replaced with a `drawablePadding` attribute.

There's a lint quickfix to perform this conversion in the Eclipse plugin.